### ISSUE TITLE
BO - Carriers > In General Tab, Current size is always undefined

### ISSUE THREAD

**Issue by florine2623** [PrestaShop/PrestaShop#36395]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

When I go to Edit a carrier page, I can't see the size of the logo I selected for my carrier. It it aways displayed as undefined.

### Expected behavior

The size should be displayed.

<img width="1202" alt="Screenshot 2024-06-20 at 15 19 52" src="https://github.com/PrestaShop/PrestaShop/assets/16019289/a791039b-df04-4f16-8f94-7b39b169a8b2">

### Steps to reproduce

1. Go to BO > Shipping > Carriers
2. Edit any carrier > Add a logo
3. See Current size is always undefined :x:

### PrestaShop version(s) where the bug happened

8.0.x, 8.1.7, develop

### PHP version(s) where the bug happened

8.1

**Comment by paulnoelcholot** [comment]
Hello @florine2623,

I reproduce the issue with PrestaShop version develop, I'll add this to the backlog so it can be fixed.

Please be aware that some issues might take a very long time to be resolved. If this one is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you. 

If you fix the issue on your end, please [contribute it back to the project](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you

![image](https://github.com/PrestaShop/PrestaShop/assets/112863083/13e4e001-6196-4841-9a88-83383fe25616)

**Comment by Hlavtox** [comment]
@PrestaShop/qa-functional Still relevant?

**Comment by boo-code** [comment]
@Hlavtox yes, still relevant - and the reason is smaller than it looks, plus there is already a fix waiting for review.

The label is a hardcoded placeholder that nothing ever replaces:

```smarty
{* admin-dev/themes/default/template/controllers/carrier_wizard/helpers/form/form.tpl:38 *}
{l s='Current size:' d='Admin.Shipping.Help'} <span id="carrier_logo_size">{l s='undefined' d='Admin.Shipping.Help'}</span>.
```

The word "undefined" there is not a JavaScript value leaking into the page - it is a **translated string** shipped as the initial content of the span, evidently meant to be overwritten once a size is known. Searching the whole tree for `carrier_logo_size` returns that single line: no JavaScript, no controller, nothing assigns it. So the span keeps its placeholder forever, and the page always reads "Current size: undefined".

Present on `8.2.x`, `9.2.x` and `develop` alike - the template line and the absent setter are identical on all three, so it has not been fixed anywhere.

**There is an open pull request for it**: #36708 by @Poulinhoo, touching that template, `logo.tpl` and `AdminCarrierWizardController.php`. It has had **no review since it was opened**, and its last activity is 2024-08-20. If the approach in it is sound, reviewing that is the shortest path here rather than a new fix - I have deliberately not written one, to avoid duplicating a contribution that is simply waiting.
